contents/blog/posthog-vs-ga4.md

```diff
@@ -0,0 +1,497 @@
+---
+date: 2023-10-12
+title: "In-depth: PostHog vs Google Analytics 4"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-vs-kubit/posthog-vs-kubit.jpg
+featuredImageType: full
+author:
+    - lior-neu-ner
+category: General
+tags:
+    - Comparisons
+    - Guides
+---
+
+Want to understand how PostHog and Google Analytics 4 (GA4) are different? Here's the short answer:
+
+- **Google Analytics** is a web analytics tool to track and analyze your website performance. It's built for marketers.
+
+- **PostHog** is an all-in-one platform that *includes* web analytics, product analytics and a whole bunch more – like session replays, A/B testing, surveys, and feature flags. It's built for engineers and product teams.
+
+In this article we’ll explore these differences in more detail, and take a deep dive into the analytics features of both.
+
+## How is PostHog different to Google Analytics?
+
+### 1. PostHog does more than analytics
+
+[Analytics](/product-analytics) is a core part of our platform, but it's just one component in a set of important tools for product teams. PostHog enables you to collect feedback with [surveys](/surveys), understand user behavior with [session replays](/session-replay), test changes with [A/B tests](/ab-testing), and deploy changes with [feature flags](/feature-flags).
```

Comment:
The ./surveys page is not live yet - waiting on  https://github.com/PostHog/posthog.com/pull/6580